packages/experimental/Avatar/ios/MSFAvatarViewManager.m

```diff
@@ -51,7 +51,8 @@ + (MSFAvatarData *)MSFAvatarData:(id)json
 
 @interface RCT_EXTERN_MODULE(MSFAvatarViewManager, RCTViewManager)
 
-RCT_EXPORT_VIEW_PROPERTY(avatarSize, MSFAvatarSize);
```

Comment:
The macro in question that needed to be updated.
This maps the JS "size" prop to the swift "avatarSize" property.

Reply:
So was "size" no-op in JS?

Reply:
It changed the frame of the view, but didn't actually set AvatarSize. This resulted in having the Avatar of a small size getting stretched out to a bigger size, like so:

![Screen Shot 2020-12-11 at 9 48 09 PM](https://user-images.githubusercontent.com/6722175/101973554-7c6a3800-3bfe-11eb-8bc6-bae0d4cca0d9.png)